src/pages/home/report/ReportActionCompose.js

```diff
@@ -399,9 +400,11 @@ class ReportActionCompose extends React.Component {
      */
     isLikeEmojiCode(str, pos) {
         const newLineOrWhiteSpaces = /[\n\s]/g;
+        const hasColumnOnlyAtTheBegin = /^:[^:]+$/;
```

Comment:
Same here, except this function is called a lot more frequently than `insertSelectedEmoji` (every keystroke)